h-youjin/LV.1/수박수박수박수박수박수.js

```diff
@@ -0,0 +1,9 @@
+function solution(n) {
+  let answerStr = '';
+
+  for (let i = 0; i < n; i++) {
+    answerStr += i % 2 === 0 ? '수' : '박';
+  }
+
+  return answerStr;
+}
```

Comment:
```suggestion
function solution(n) {
  let answerStr = '';

  for (let i = 0; i < n; i++) {
    answerStr += i % 2 === 0 ? '수' : '박';
  }

  return answerStr;
}
```
저는 if - else문 사용했지만 유진님은 삼항연산자를 사용해서 더 간결하게 작성하셨네요!!! 😁
저도 앞으로 좀 더 간단하게 작성하도록 노력해봐야겠어요!!👍
항상 좋은 코드 잘 보고 갑니다~!!  😊

Reply:
감사합니다 유진님 ! 저도 항상 유진님 코드 잘 보고있습니다 :)